Documentation/Books/AQL/Operations/Insert.md

```diff
@@ -100,6 +100,22 @@ FOR i IN 1..1000
   } INTO users OPTIONS { overwrite: true }
 ```
 
+
+In contrast to the MMFiles engine, the RocksDB engine does not require collection-level
+locks. Different write operations on the same collection do not block each other, as
+long as there are no _write-write conficts_ on the same documents. From an application
+development perspective it can be desired to have exclusive access on collections,
+to simplify the development. 
+Exclusive access can also speed up modification querys, because we avoid conflict checks.
```

Comment:
If you really want a line break here, add `<br/>` at the end of line 108. If you want to start a new paragraph, insert an empty line above.

I think there should be a an additional line break above line 109, and the one on line 110 can be removed.